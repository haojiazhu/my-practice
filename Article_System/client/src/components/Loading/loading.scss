.loading {
	align-items: center;
	display: flex;
	height: 100%;
	justify-content: center;
	position: absolute;
	width: 100%;
	z-index: 3;
}

.cover {
	background: rgba(66, 66, 66, 0.95);
    height: 100%;
	opacity: 0.8;
	position: absolute;
	width: 100%;
	z-index: 2
}
