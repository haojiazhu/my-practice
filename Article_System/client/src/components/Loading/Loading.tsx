import React, { Component } from 'react';
import './loading.scss';

const Loading: any = () => {
	return (
		<div>
			<div className="cover" />
			<div className='loading'>
				<div className="spinner-border text-light" role="status">
  					<span className="sr-only">Loading...</span>
				</div>
			</div>
		</div>
	)
}

export default Loading;
