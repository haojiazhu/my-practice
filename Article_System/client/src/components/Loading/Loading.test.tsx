import React from 'react';
import Loading from './Loading';
import renderer from 'react-test-renderer';

it('render loading', () => {
	const loading = renderer.create(<Loading />).toJSON();
	expect(loading).toMatchSnapshot();
});
