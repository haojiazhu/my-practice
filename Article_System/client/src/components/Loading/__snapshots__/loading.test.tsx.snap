// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render loading 1`] = `
<div>
  <div
    className="cover"
  />
  <div
    className="loading"
  >
    <div
      className="spinner-border text-light"
      role="status"
    >
      <span
        className="sr-only"
      >
        Loading...
      </span>
    </div>
  </div>
</div>
`;
